engine/src/flutter/tools/licenses_cpp/src/catalog.cc

```diff
@@ -17,6 +17,35 @@ bool Overlaps(std::string_view a, std::string_view b) {
 
   return start1 < end2 && start2 < end1;
 }
+
+bool EndsWith(std::string_view str, std::string_view suffix) {
```

Comment:
std::string_view has "ends_with"

https://en.cppreference.com/w/cpp/string/basic_string/ends_with.html

Reply:
since C++20, do we have c++20 now?

Reply:
Are you asking if internally the company uses C++20? Yes.

Reply:
ignored: our project hasn't fully moved to c++20 yet.

Reply:
(we're on c++17 still)